client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -19,7 +19,7 @@ import {
   DisplayTableDownloadCsv,
   DisplayTableColumnToggle,
   PageSelector,
-  PageinateBySelector,
+  SelectPageSize,
```

Comment:
Minor point, "page size" seemed generally a more accurate name than "paginate by". Conveys the concept more directly, and avoids the connotation of `___By` with methods (like `countBy`, `groupBy`, etc.)